### ISSUE TITLE
Adding systems to one of the stages in a `FixedTimestepStage`?

### ISSUE THREAD

**Issue by johanbluecreek** [IyesGames/iyes_loopless#38]
TL;DR: Is there a way to add system sets to the stages of a `FixedTimestepStage` that has been added to my `App` elsewhere?

How I understand the documentation one adds the `FixedTimestepStage` into the scheduler, e.g. before the `CoreStage::Update`. Then `FixedTimestepStage` has sub-stages which are executed at the time-step. In a sub-stage, I can use `label`s to order the execution of my systems using `before` or `after`. But say I want to make a plugin that adds a system set to one of these sub-stages, to be able to have access to these labels, how do I do that?

More concretely: I'm looking after a design in which I can have a plugin handling `A` components, one plugin handling `B`, and then one plugin handling their interaction. The interaction should take place after `A` and `B` have both been updated, so I can label the `ConditionSet`s I have defined for `A` and `B`, but to have access to those labels it appears that all systems needs to be part of the same sub-stage. That is I imagine to be able to do something along these lines

```rust
impl Plugin for InitFixedTimestepStage {
    fn build(&self, app: &mut App) {
        app.add_stage_before(
            CoreStage::Update,
            "ft_label",
            FixedTimestepStage::new(Duration::from_secs_f32(TIME_STEP))
                .add_stage_with_label( // suggested method
                    "interaction_label",
                    SystemStage::parallel() // adding empty stage to add systems to later
                )
        )
    }
}

impl Plugin for UpdateComponentA {
    fn build(&self, app: &mut App) {
        app.schedule
            .get_stage_mut("ft_label").unwrap()
            .get_stage_mut("interaction_label").unwrap() // suggested method
            .add_system_set(
                ConditionSet::new()
                    .label("update_A")
                    .with_system(update_A)
                    .into()
            );
    }
}

// same for B... then:

impl Plugin for InteractionAB {
    fn build(&self, app: &mut App) {
        app.schedule
            .get_stage_mut("ft_label").unwrap()
            .get_stage_mut("interaction_label").unwrap() // suggested method
            .add_system_set(
                ConditionSet::new()
                    .after("update_A") // This label...
                    .after("update_B") // ... and this label is now available as we are in stage "interaction_label"
                    .label("interact_AB")
                    .with_system(interact_AB)
                    .into()
            );
    }
}
```

This would make the plugins more modular, otherwise I'd have to design large plugins that has access to all systems at once.

Or is there a way to do this that I'm just missing?
